### DISCUSSION TITLE
获取文件列表请求`/api/fs/list`响应较慢

### DISCUSSION THREAD

**Discussion by LockonS** [AlistGo/alist#5885] [General]
获取文件列表请求`/api/fs/list`响应时间稍有点长，大约需要800ms。
首先不确定这个是否是一个问题，可能只是个人觉得有点慢。不知道是否只有我遇到了同样的情况。

Alist版本：3.30.0
基本上可以排除是网络延迟问题。在云服务器和本地服务器都有配置Alist V3作为文件传输工具，配置的存储类型是本地存储，两个实例下`/api/fs/list`请求的响应时间都达到了750ms到800ms之间。
配置不足问题也可以排除。云服务器是2核4GB，本地服务器是8核16GB，同时在跑的其他服务也比较少。

**Comment by xhofe** [comment]
![CleanShot 2024-01-14 at 13 25 44@2x](https://github.com/alist-org/alist/assets/36558727/5245de53-039b-4d84-8e68-2e0935ece5a7)

**Reply by LockonS** [reply]
尝试了多次，大部分时间都在40ms以下

<img width="1102" alt="截屏2024-01-14 14 33 23" src="https://github.com/alist-org/alist/assets/5363950/c11a3ea7-6255-46b9-b359-487a450420c1">

在Chrome里显示的 Content Download 时间比较长，应该就是这里了，还需要再研究一下什么情况😂
<img width="419" alt="截屏2024-01-14 14 34 56" src="https://github.com/alist-org/alist/assets/5363950/b3d0605d-d604-49a5-83fb-132eaadd2cec">

顺便切换到Safari里测试了一下，非常正常
<img width="1216" alt="截屏2024-01-14 14 38 26" src="https://github.com/alist-org/alist/assets/5363950/2f24a00c-1ec5-4115-874f-da6fb9ea89e7">

**Comment by LockonS** [comment]
鉴于两个浏览器的巨大差异，初步怀疑是浏览器插件的问题，尝试性先把AdBlock对自己的Alist加了白名单。
然后就Content Download就降到了1ms左右。

感谢大佬的回复~！

<img width="397" alt="截屏2024-01-14 14 48 15" src="https://github.com/alist-org/alist/assets/5363950/ae0db2b0-9a15-436a-999b-1041396e9bef">
